custom_components/tahoma/const.py

```diff
@@ -99,7 +71,6 @@
     "Screen": COVER,
     "Shutter": COVER,
     "Siren": SWITCH,
-    "SirenStatus": None,  # widgetName, uiClass is Siren (switch)
```

Comment:
We need to keep this, otherwise it will fallback to Switch (since uiClass siren is mapped to Switch).

Reply:
It seems to be an hack. Can we fix it in someway?

Reply:
I am not sure. It is an exception because Somfy uses the same ui class 🥺.